pkg/arm/data/master-startup.sh

```diff
@@ -288,6 +290,11 @@ servingInfo:
   keyFile: master.server.key
   maxRequestsInFlight: 500
   requestTimeoutSeconds: 3600
+  namedCertificates:
```

Comment:
When web-console will be officially accessed via route we will remove this and move to certs inside the secret mode